psl/psl/tests/common/mod.rs

```diff
@@ -10,6 +10,7 @@ pub(crate) fn reformat(input: &str) -> String {
     psl::reformat(input, 2).unwrap_or_else(|| input.to_owned())
 }
 
+#[track_caller]
```

Comment:
Still useful?

Reply:
Yes, this kind of helper/assertion function definitely should have `#[track_caller]`. It doesn't make sense to remove it just to put it back manually each time you investigate a failing test.